utilities/dbviz/Earthfile

```diff
@@ -33,13 +35,16 @@ all-hosts-check:
 build:
     FROM +builder
  
-    RUN /scripts/std_build.py  --bins="dbviz/dbviz"
+    RUN echo cache breaker 1
```

Comment:
Why do we need this ?

Reply:
We don't its debug code that I missed removing.